helmfile/charts/gatekeeper/templates/policies/user-crds.rego

```diff
@@ -0,0 +1,72 @@
+package k8susercrds
+
+# Check when there are no valid reviews
+violation[{"msg": msg}] {
+    review := fetchFaildReviews[_]
```

Comment:
This looks like a typo, fix for the whole file. 
```suggestion
    review := fetchFailedReviews[_]
```

Reply:
Yeah i guess i can drop the output review. I used it as i knew it was the correct one.